guide/daily-spending-wallet/first-use.md

```diff
@@ -35,102 +69,133 @@ Illustration sources
-   retina = "/assets/images/guide/daily-spending-wallet/first-use/product-intro-2@2x.png"
-   alt-text = "Introduction screen explaining recovery"
-   caption = "Talk about recovery"
-   layout = "shadow"
-   width = 250
-   height = 541
-%}
+Regularly [backing up wallet]({{ '/guide/daily-spending-wallet/backup-and-recovery/landing-page/' | relative_url }}) and user data in a secure manner is essential for bitcoin applications. For small amounts of funds, [encrypted cloud backups]({{ '/guide/daily-spending-wallet/backup-and-recovery/cloud-backup/' | relative_url }}) are a convenient choice with a good tradeoff between security and convenience.
 
-{% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/first-use/product-intro-3.png"
-   retina = "/assets/images/guide/daily-spending-wallet/first-use/product-intro-3@2x.png"
-   alt-text = "Introduction screen explaining security"
-   caption = "Talk about security"
-   layout = "shadow"
+Frequent backups are important for lightning wallets, as channel states frequently change and need to be updated.
+
+</div>
+
+## Security setup
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/daily-spending-wallet/first-use/security-setup.png"
+   retina = "/assets/images/guide/daily-spending-wallet/first-use/security-setup@2x.png"
    width = 250
    height = 541
+   alt-text = ""
+   caption = "Additional security measures are appropriate since bitcoin wallets handle user funds."
+   layout = "float-right-desktop -background -shadow"
 %}
 
+While smartphones typically already require authentication to get past their lock screens, it is appropriate for bitcoin applications to introduce further security measures. For example, a user may only require [Touch ID or Face ID]({{ '/guide/daily-spending-wallet/security/#biometrics-touch-id--face-id' | relative_url }}) to access their device, but then set up a [unique PIN]({{ '/guide/daily-spending-wallet/security/#pin-protection' | relative_url }}) to access the wallet and send payments.
+
+PINs can also be used to encrypt backup data, both locally and in cloud storage.
+
 </div>
 
- These screens lay out the unique features of self-custodial wallets to make sure the user understands these caveats. These screens suggest talking about ownership, explicitly stating the recovery mechanism that the respective wallet uses, and security measures that they should take to protect their wallet.
+## Initial deposit
+
+Any new wallet starts empty and requires funding to become useful, typically via purchase of bitcoin, or deposit of already owned bitcoin. Both on-chain and lightning deposits should be supported, with on-chain funds automatically getting swapped to lightning. Swap fees should be explained in advance to prevent user frustration. For more on this, see [funding]({{ '/guide/daily-spending-wallet/funding/' | relative_url }}).
```

Comment:
```suggestion
New wallets start empty and require funding to become useful. Typically, bitcoin is purchased and sent from an exchange, or from another wallet the user controls. Both on-chain and lightning deposits should be supported. On-chain deposits should be automatically [swapped]({{ '/guide/how-it-works/lightning-services/#swaps' | relative_url }}) to lightning. To prevent user frustration, swap fees should be explained when creating the deposit request. For more on this, see [funding]({{ '/guide/daily-spending-wallet/funding/' | relative_url }}).
```
Minor tweaks on top of your edits.

Don't think explaining the unified payment request thing is important right here, that can be covered in the funding page.